playbooks/bibdata.yml

```diff
@@ -0,0 +1,26 @@
+---
+# by default this playbook runs in the staging environment
+# to run in qa, pass '-e runtime_env=qa'
+# to run in production, pass '-e runtime_env=production'
+- name: Build bibdata infrastructure
+  hosts: bibdata_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../site_vars.yml
+    - ../group_vars/bibdata/vault.yml
+    - ../group_vars/bibdata/{{ runtime_env | default('staging') }}.yml
+  roles:
+    - role: roles/bibdata
+    - role: roles/bibdata_sqs_poller
+    - role: roles/sidekiq_worker
+      when: "'worker' in inventory_hostname"
+    - role: 'hr_share'
+    - role: roles/datadog
+      when: runtime_env == "production"
```

Comment:
@acozine I really hate to delay this PR any longer, but I noticed yesterday that this line causes issue when runtime_env is not set in the approvals playbook.  I created a PR for approvals to fix it (#2910)  Would you prefer to merge this PR and ticket the fact that it will not run without the runtime_env being set, or did you want to update it prior to me merging?